### ISSUE TITLE
No Passed Filter variant

### ISSUE THREAD

**Issue by abrusell** [exomiser/Exomiser#536]
Hi,
I've been using exomiser to analyse WGS data for a long time, and it's the first time I notice this behaviour:
with some HPO IDs (in particular HP:0002597 and/or HP:0001626) I get 0 passed variant filters...

`Loaded 100000 variants - 0 passed variant filters...
Loaded 200000 variants - 0 passed variant filters...
Loaded 300000 variants - 0 passed variant filters...
Loaded 400000 variants - 0 passed variant filters...
Loaded 500000 variants - 0 passed variant filters...
Loaded 600000 variants - 0 passed variant filters...
Loaded 700000 variants - 0 passed variant filters...
Loaded 800000 variants - 0 passed variant filters...
Processed 720095 variant records into 824420 single allele variants (including 0 structural variants)
Variant annotation finished in 0m 32s 321ms (32321 ms)
Loaded 824420 variants - 0 passed variant filters
Checking inheritance mode compatibility with [AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL] for genes which passed filters
Running GeneFilter: InheritanceFilter{compatibleModes=[AUTOSOMAL_DOMINANT, AUTOSOMAL_RECESSIVE, X_RECESSIVE, X_DOMINANT, MITOCHONDRIAL]}
Running Prioritiser: OmimPrioritiser{}
Variant filter stats are:
PRIORITY_SCORE_FILTER: pass=0 fail=38347
QueryPhenotypeMatch{organism=HUMAN, bestPhenotypeMatches=[], bestMatchScore=0.0, bestAverageScore=0.0} has no phenotype matches for input set [PhenotypeTerm{id=HP:0002597, label=Abnormality of the vasculature, present=true}]`

This doesn't happen with different HPO IDs 
May it be a bug?

**Comment by julesjacobsen** [comment]
Can you include the analysis steps you were running, please.  These two terms are very high up the ontology tree and will therefore have a very low information content (IC) and hence might have been excluded from the database. If you use more informative terms you should see much more informative results.

e.g. child terms of Abnormal cardiovascular system morphology ([HP:0030680](https://hpo.jax.org/app/browse/term/HP:0030680)) should give you some results.
